easybuild/easyconfigs/f/FastEpistasis/FastEpistasis-2.07-foss-2018a.eb

```diff
@@ -0,0 +1,37 @@
+#
+# This file is an EasyBuild reciPY as per github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2021 University of Geneva Switzerland
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+#
+easyblock = 'CMakeMake'
+
+name = 'FastEpistasis'
+
+version = '2.07'
+
+homepage = 'https://gitlab.sib.swiss/tschuepb/FastEpistasis'
+
+description = '''
+   FastEpistasis, a software tool capable of computing tests of epistasis for a large number of SNP pairs, is an 
+   efficient parallel extension to the PLINK epistasis module.'''
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://gitlab.sib.swiss/tschuepb/FastEpistasis/uploads/864695cfa08720b46dd1b437faca3fff/']
+sources = ['%(name)s-%(version)s.tar.bz2']
+checksums = ['3751461a0c410749ce77b07b957f7e9c4cbfb18aa4003b5aab3fc52db10b5f33']
+
+dependencies = [
+    ('CMake', '3.10.2'),
```

Comment:
@ysagon The compilation problem I described is resolved by downgrading `CMake` to `3.9.1` (the required easyconfig already exists)...